source/fundamentals/transactions.txt

```diff
@@ -45,313 +45,240 @@ you want to execute sequentially. We recommend you reuse
-   see the
-   :manual:`Create Collections and Indexes in a Transaction </core/transactions/#std-label-transactions-create-collections-indexes>`
-   server guide.
+Transaction Options
+-------------------
 
-.. _nodejs-transaction-core-api-example:
+You can pass a ``TransactionOptions`` instance to the
+``startTransaction()`` and ``withTransaction()`` methods to configure
+how the driver performs a transaction. When you specify an option,
+it overrides the value of the option that you might have set on your
+``MongoClient`` instance.
 
-Core API Implementation
-~~~~~~~~~~~~~~~~~~~~~~~
+The following table includes options that you can specify
+in a ``TransactionOptions`` instance:
 
-The code example in this section demonstrates how you can use the Core API to
-run the multi-document payment transaction in a session. This function
-shows how you can perform the following:
+.. list-table::
+   :widths: 25 75
+   :header-rows: 1
 
-1. Start a session
-#. Start a transaction, specifying transaction options
-#. Perform data operations in the same session
-#. Commit a transaction, or cancel it if the driver encounters an error
-#. End a session
+   * - Setting
+     - Description
 
-.. literalinclude:: /code-snippets/transactions/txn-core.js
-   :language: javascript
-   :linenos:
-   :emphasize-lines: 3-7,13,24,40,51,61,65,73-74,76-77,81,84
-   :start-after: start placeOrder
-   :end-before: end placeOrder
+   * - ``readConcern``
+     - | Specifies read operation consistency of the replica set.
+       | To learn more, see :manual:`Read Concern
+         </reference/read-concern/>` in the Server manual.
 
-You must pass the session object to each CRUD operation that
-you want to run on that session.
+   * - ``writeConcern``
+     - | Specifies the write operation level of acknowledgment required
+         from a replica set.
+       | To learn more, see :manual:`Write Concern
+         </reference/write-concern/>` in the Server manual.
 
-.. important:: Use a Session with the Client That Started It
+   * - ``readPreference``
+     - | Specifies how to route read operations to members of a replica set.
+       | To learn more, see :manual:`Read Preference
+         </core/read-preference/#read-preference>` in the Server manual.
 
-   Starting in version 6.0 of the {+driver-short+}, the driver
-   throws an error if you provide a session from one ``MongoClient``
-   instance to a different client instance.
+   * - ``maxCommitTimeMS``
+     - Specifies the length of time that a commit action on a
+       transaction can run, in milliseconds.
 
-   For example, the following code generates an
-   ``MongoInvalidArgumentError`` error because it creates
-   a ``ClientSession`` instance from the ``client1`` client, but provides
-   this session to the ``client2`` client for a write operation:
+For a full list of options, see the API documentation for
+`TransactionOptions <{+api}/interfaces/TransactionOptions.html>`__.
```

Comment:
Issue:
The api source constant is malformed.

Suggestion:
I think `{+api+}` could fix the issue.